dbms/src/Core/SpillHandler.h

```diff
@@ -55,8 +56,10 @@ class SpillHandler
     size_t partition_id;
     Int64 current_spilled_file_index;
```

Comment:
need initialize current_spilled_file_index and current_append_write in constructor, others LGTM.

Reply:
They are inited in `setUpNextSpilledFile`, which is called in the constructor.